stable/kommander/Chart.yaml

```diff
@@ -1,10 +1,10 @@
 apiVersion: v1
-appVersion: 1.184.0
+appVersion: 1.0.0
```

Comment:
You are effectively decreasing the appVersion here. Do you really want to do that?

Reply:
is already happened in https://github.com/mesosphere/charts/pull/375 - i just started this branch before the other PR got merged. and yes, we do want to do it. appVersion seems to be metadata and should have never been pinned to UI version. from that I understand this is going to be our customer facing version for kommander.